docs/modules/ROOT/pages/governance.adoc

```diff
@@ -197,3 +202,229 @@ pub trait TimelockABI<TState> {
     fn renounceRole(ref self: TState, role: felt252, account: ContractAddress);
 }
 ----
+
+== Votes
+
+The {votes-component} provides a flexible system for tracking voting power and delegation. This system allows users to delegate their voting power to other addresses, enabling more active participation in governance.
```

Comment:
```suggestion
The {votes-component} provides a flexible system for tracking voting power. This system allows users to delegate their voting power to other addresses, enabling more active participation in governance.
```

Reply:
Same as above. In the comment section of the component itself we are including "The Votes component provides a flexible system for tracking and delegating voting power..." and `delegate`and `delegate_by_sig`are two important functions of the component

Reply:
The issue is not mentioning delegation, it is a matter of phrasing (@andrew-fleming can help here since I'm not a native speaker). To me "The Votes component provides a flexible system for tracking and delegating voting power" sounds ok, while "flexible system for tracking voting power and delegation" sounds weird, since the second sentence seems to imply that delegation, similar to voting power, is also tracked. Rephrasing the sentence could help, but just removing "and delegation" seemed ok, since that is also mentioned in the next sentence.

Reply:
> To me "The Votes component provides a flexible system for tracking and delegating voting power" sounds ok, while "flexible system for tracking voting power and delegation" sounds weird, since the second sentence seems to imply that delegation, similar to voting power, is also tracked

Hmm yeah, I see what you mean. Even in a super obvious context like: "...provides a flexible system for eating pizza and delegation," the system obviously isn't for eating delegation, but IMO consistency with listed items offers a more clarified expression

Reply:
Went with "The {votes-component} provides a flexible system for tracking and delegating voting power. This system allows users to delegate their voting power to other addresses, enabling more active participation in governance." and the same for the above comment. Hope everyone is happy with that one.